.git-authors

```diff
@@ -5,6 +5,9 @@ pairs:
   ym: Yugma Moradia; yugma93
   kj: Karthi Jagatheesan
   km: Klaus Mana
+  nw: Nathan Wang; nathan.wang2
+  sc: Serena Cao; wcao0630
+
```

Comment:
This is weird. This is already part of master and not part of your changes. I would recommend you apply your changes on top of what's on master. I believe the only change you would make is in the `ListOfProjects.css` file.

Reply:
I know. I made changes in `ListOfProjects.css` and nowhere else.